src/smart-app-banner/smart-app-banner.styles.ts

```diff
@@ -0,0 +1,85 @@
+import { Text, TextStyleHelper } from "../text";
+import { Button } from "../button";
+import { Color } from "../color";
+import { MediaQuery } from "../media";
+import styled from "styled-components";
+import { IconButton } from "../icon-button";
+import { Icon } from "../icon";
+
+export const SmartAppBannerContainer = styled.div<{ $offset: number }>`
+    display: flex;
+    flex-direction: row;
+    align-items: stretch;
+    position: fixed;
+    margin: 8px 20px;
```

Comment:
can we use `rem` instead of `px`? this applies to other styles as well

Reply:
okay, changed! but for top, because we expect to pass in offset as a calculated value in pixels, I have left it in px